plugins/quota/init.go

```diff
@@ -0,0 +1,80 @@
+package quota
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "quota"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "QuotaEnable",
+			Method:      "POST",
+			Pattern:     "/quota/{volname}",
+			Version:     1,
+			HandlerFunc: quotaEnableHandler},
+		route.Route{
+			Name:        "QuotaDisable",
+			Method:      "DELETE",
+			Pattern:     "/quota/{volname}",
+			Version:     1,
+			HandlerFunc: quotaDisableHandler},
+		route.Route{
+			Name:        "QuotaList",
```

Comment:
You dont need many APIs. You can merge both size and inode quota as a single API - one to set, one to list, one to remove.

Is it more expensive to get both information (size and inode quota) from backend ?

Reply:
There is no additional cost in setting / getting / removing it.
I think it is a good idea to merge the two, though directory quota alone is the more stable and widely used option.
@aravindavk  Do u agree in it be merged into single API.

Reply:
+1